rules/parser.y

```diff
@@ -235,6 +237,28 @@ extra_labels_opts  :
                      { $$ = true }
                    ;
 
+vector_matching    :
+                     { $$ = nil }
+                   | GROUP_OP '(' label_list ')'
+                     { 
+                       var err error
+                       $$, err = newVectorMatching("", $3, nil)
+                       if err != nil { yylex.Error(err.Error()); return 1 }
+                     }
+                   | GROUP_MOD GROUP_OP '(' label_list ')'
```

Comment:
```
a{l="x"} 1                            c{l="x"} 1
b{l="x"} 2           *  by(l)         c{l="y"} 2
a{l="y"} 4
```

Will fail without `group_left` due to `c{l="x"}` matching twice on the left. With it resolves two entries having the same labelset - so the output is arguably useless.